static/progressive-web-app.js

```diff
@@ -47,6 +47,25 @@ self.addEventListener("activate", (event) => {
 self.addEventListener("fetch", (event) => {
   const url = new URL(event.request.url);
 
+  // Network-first strategy for index.html and dynamic routes
+  if (url.origin === location.origin && !urlsToCache.includes(url.pathname)) {
+    event.respondWith(
+      fetch(event.request)
+        .then((networkResponse) => {
```

Comment:
Can you change this to async await syntax

Reply:
will fix that once #136 goes through, since it changes PWA a lil bit

Reply:
Okay merged!